Mathlib/CategoryTheory/Abelian/InjectiveResolution.lean

```diff
@@ -69,7 +69,7 @@ lemma exact₀ {Z : C} (I : InjectiveResolution Z) :
 def descFOne {Y Z : C} (f : Z ⟶ Y) (I : InjectiveResolution Y) (J : InjectiveResolution Z) :
     J.cocomplex.X 1 ⟶ I.cocomplex.X 1 :=
   J.exact₀.descToInjective (descFZero f I J ≫ I.cocomplex.d 0 1)
-    (by dsimp; simp [← assoc, descFZero])
+    (by dsimp; simp [← assoc, assoc, descFZero])
```

Comment:
What's up with `assoc`ing in both directions? Is this explained somewhere already?

Reply:
See https://github.com/leanprover/lean4-nightly/releases/tag/nightly-2024-06-05, and in particular https://github.com/leanprover/lean4/pull/4345